examples/multithreaded.rs

```diff
@@ -66,7 +66,7 @@ fn main() {
                         let state = !modifiers.shift();
                         use VirtualKeyCode::*;
                         match key {
-                            A => window.set_always_on_top(state),
+                            A => window.set_window_level(WindowLevel::AlwaysOnBottom),
```

Comment:
maybe `Key1 => AlwaysOnTop`, `Key2 => AlwaysOnBottom` and `Key3 => Normal`
allowing to set all 3 states individually would be good